zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -255,3 +269,48 @@ def get_parse_product_request(
         )
         scrapy_request.meta["allow_offsite"] = True
         return scrapy_request
+
+
+class ExperimentalEcommerceSpiderParams(BaseModel):
+    urls: List[str] = Field(
```

Comment:
I'm good with this name. 👍  Although we might want to put it somewhere else where the upcoming Article spider by @PyExplorer can readily use it as well.

Reply:
> Although we might want to put it somewhere else where the upcoming Article spider by @PyExplorer can readily use it as well.

Related: https://github.com/zytedata/zyte-spider-templates/pull/34